components/UpSellPlusModal.props.ts

```diff
@@ -0,0 +1,20 @@
+export interface UpSellPlusModalProps {
```

Comment:
Upsell is one word 😄
```suggestion
export interface UpsellPlusModalProps {
```

Reply:
[✏️ Upsell is one word!](https://github.com/likecoin/3ook-com/pull/239/commits/cbba48767bbe58dabb06d3ef6fca56b0852ca36b)